### PR TITLE
Feature/oni 132

### PR DESCRIPTION
**Pull request by wzglinieckisoldevelo**

### CODE REVIEW

File: claim/gql_mutations.py [hunk 1]
```diff
@@ -369,6 +369,7 @@ def async_mutate(cls, user, **data):
                 return [{
                     'message': _("claim.mutation.duplicated_claim_code") % {'code': data['code']},
                 }]
+            is_claim_code_autogenerated = data.get("autogenerate_code", None)
```
Comment (malinowskikam) [inline claim/gql_mutations.py:372]: The default value for boolean field should be `False`. `None` works too, but it does't indicate the actual type of the field

File: claim/gql_mutations.py [hunk 2]
```diff
@@ -377,6 +378,8 @@ def async_mutate(cls, user, **data):
             claim = update_or_create_claim(data, user)
             if attachments:
                 create_attachments(claim.id, attachments)
+            if is_claim_code_autogenerated:
+                return {"client_mutation_label": f"Create Claim - {claim.code}", "code": f"{claim.code}"}
```
Comment (malinowskikam) [inline claim/gql_mutations.py:382]: We need a proper way of messaging between the mutation code and the mutation handling on core. I understand this solution, but i don't like it in a long term.

**Review by malinowskikam — APPROVED** [review]
Looks good to me, a left some comments, but nothing blocking

**Comment by dragos-dobre** [comment]
Changes already merged. Closing the ticket.

**Pull request closed without merging**